### ISSUE TITLE
Without category

### ISSUE THREAD

**Issue by ivansglazunov** [lob/generate-changelog#7]
I can pull request to add support commits without category?

```
git commit -m "feat: some text 1"
git commit -m "feat(): some text 2"
git commit -m "feat(abc): some text 3"
```

``` md
#### 0.0.1 (2016-9-22)

##### New Features

* some text 1 (7cd910d0)
* some text 2 (7cd910d0)
* **abc:** some text 3 (7cd910d0)
```

**Comment by tvardy** [comment]
Isn't that closed by #16 ?

**Comment by ivansglazunov** [comment]
Yes, thanks.
